### ISSUE TITLE
ERROR: Something went wrong while compiling ROOT!

### ISSUE THREAD

**Issue by mattmar2410** [zoglauer/megalib#27]
Hi Andreas,

I am having a similar issue to Sylvain about a year ago. I know you suggested that if you have installed homebrew, it should be commented out in your .bash_profile file, because having unexpected libraries in your path is usually the most common reason for ROOT failing to compile. However I do not know how to do this. 

My entry looks like this: 

megalib Matt$ bash setup.sh --release=dev --patch=yes --clean=yes

I attached my error message. What do you recommend I do? 

Any help is appreciated! 

Thanks,
Matt

<img width="634" alt="screen shot 2017-10-27 at 3 50 22 pm" src="https://user-images.githubusercontent.com/33163810/32128832-2ab22abe-bb36-11e7-9a47-daa0bdf2a5e1.png">

**Comment by jccurtis** [comment]
@zoglauer, 

I see that you fixed this issue with the current experimental branch in 974a9e8 by updating ROOT to 6.10. I just tested on macOS 10.13. You have also added several other features in the experimental branch - 689 commits ahead of `master`. What is the current limitation with making the next release? The current `master`:`setup.sh` doesn't use the experimental release so it took me a while to resolve this issue which you fixed on Friday 😛 

Let me know if I can do anything to help with the release! We have a group of grad students at LBL that will be using the package for simulations in the coming months.  

FYI @mattmar2410 @margobatie

**Comment by zoglauer** [comment]
I did not know I fixed the issue - I just updated MEGAlib to work with the lastest macOS & xcode

At the moment all developments happen in the experimental branch which include all the improvements for the analysis of the MEGAlib balloon flight as well as for the AMEGO mission. This branch should be mostly stable since we use it for all our science analysis unless you delve into the latest more advanced neural network based Compton event reconstruction and advanced image reconstruction tools. There are at the moment just too many parallel developments going on that I cannot declare a stable tested version for release.

But I'll update the master to work also with ROOT 6.10.
